datadog.go

```diff
@@ -22,6 +25,18 @@ func init() {
 	log.SetOutput(os.Stderr)
 }
 
+func expandPath(path string) string {
```

Comment:
Actually I just realized why do we need to expand the path in the first place? The posix layer should convert that automatically when run.